int32[] bins_to_ignore
---
